pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -475,7 +476,6 @@ func (configgen *ConfigGeneratorImpl) buildInboundClusters(env *model.Environmen
 			// When building an inbound cluster for the ingress listener, we take the defaultEndpoint specified
 			// by the user and parse it into host:port or a unix domain socket
 			// The default endpoint can be 127.0.0.1:port or :port or unix domain socket
-			endpointAddress := actualLocalHost
```

Comment:
I would not touch this piece. Let this be as is. Because the api contract with the user is that the default endpoint IP can only be 127.0.0.1.. and nothing else.